predicate/cli/tests/test_util.py

```diff
@@ -0,0 +1,100 @@
+import os
+from jinja2 import Environment, BaseLoader
+
+
+
+from ..util import create_dir_if_not_exist, create_policy_from_template, get_classname
+
+TEST_DIR_PATH = "testpath"
+TEST_POLICY = "devOps"
+
+def test_create_dir_if_not_exist(tmp_path):
```

Comment:
should this use the argument it takes?

Reply:
Nope. Resolved this in following commit.